src/Orckestra.Composer.Website/App_Code/ImageHelpers.cshtml

```diff
@@ -30,19 +31,17 @@
                 }
             }
         }
-        <img src="@AutoImageResizingHelper.GetResizedImageUrl(Image, maxWidth > 0 ? (int?)maxWidth : null)" alt="@Image.Data.Title" loading="lazy" />
+        @{ var title = string.Empty; if (string.IsNullOrEmpty(Image.Data.Title)) { title = string.Format("title='{0}'", Image.Data.Title); }
```

Comment:
It shouldn't work - it should be `if(!string.IsNullOrEmpty(...))`.  Please test both with empty and not empty values.
 
Also - if you have a single quote character (') it will generate an invalid xml and crash..

An alternative:
```
@{ 
var imageData = Image.Data;
var title = imageData.Title;
var altText = imageData.Description;
}

<img src="@AutoImageResizingHelper.GetResizedImageUrl(Image, maxWidth > 0 ? (int?)maxWidth : null)" 
@if (!string.IsNullOrWhitespace(title)) {<text> title="@title" </text>}
@if (!string.IsNullOrWhitespace(altText)) {<text> alt="@altText" </text>}
loading="lazy" />
```